environment.sh

```diff
@@ -1,14 +1,14 @@
 export PORT=8002
 export SETTINGS='config.DevelopmentConfig'
-export PREMISES_REGISTER='http://premises.openregister.org'
-export COMPANY_REGISTER='http://company.openregister.org'
+export PREMISES_REGISTER='http://premises.alpha.openregister.org'
+export COMPANY_REGISTER='http://company.alpha.openregister.org'
 export ADDRESS_REGISTER='http://address.openregister.org'
-export LOCATION_REGISTER='http://location.openregister.org'
-export INDUSTRY_REGISTER='http://industry.openregister.org'
+export LOCATION_REGISTER='http://location.alpha.openregister.org'
+export INDUSTRY_REGISTER='http://industry.alpha.openregister.org'
 export POSTCODE_REGISTER='http://postcode.openregister.org'
-export FOOD_PREMISES_REGISTER='http://food-premises.openregister.org'
-export LOCAL_AUTHORITY_REGISTER='http://local-authority.openregister.org'
-export FOOD_PREMISES_RATING_REGISTER='http://food-premises-rating.openregister.org'
+export FOOD_PREMISES_REGISTER='http://food-premises.alpha.openregister.org'
+export LOCAL_AUTHORITY_REGISTER='http://local-authority.alpha.openregister.org'
+export FOOD_PREMISES_RATING_REGISTER='http://food-premises-rating.alpha.openregister.org'
```

Comment:
looks like this doesn't work yet with the data in the new alpha register. can you check locally by visiting `http://localhost:8002/rating/7593322015-08-27`?